lib/Doctrine/DBAL/Migrations/Version.php

```diff
@@ -404,10 +404,10 @@ private function executeRegisteredSql($dryRun = false, $timeAllQueries = false)
                     $queryStart = microtime(true);
 
                     if ( ! isset($this->params[$key])) {
-                        $this->outputWriter->write('     <comment>-></comment> ' . $query);
+                        $this->outputSqlQuery($key, $query);
                         $this->connection->executeQuery($query);
                     } else {
-                        $this->outputWriter->write(sprintf('    <comment>-</comment> %s (with parameters)', $query));
+                        $this->outputSqlQuery($key, $query);
```

Comment:
The calls to `$this->outputSqlQuery` could be moved above the `if/else` block here.